cloudpickle/cloudpickle.py

```diff
@@ -141,8 +143,22 @@ def _whichmodule(obj, name):
     return None
 
 
+if sys.version_info[:2] < (3, 7):
+    def _get_module_attr(obj):
+        if isinstance(obj, typing.TypeVar):
+            return None
+        return getattr(obj, '__module__', None)
+else:
+    def _get_module_attr(obj):
+        return getattr(obj, '__module__', None)
+
+
 def _is_global(obj, name=None):
     """Determine if obj can be pickled as attribute of a file-backed module"""
+    return _get_module_if_global(obj, name)
```

Comment:
Shouldn't this be something like the following instead?

```suggestion
    return _get_module_if_global(obj, name) is not None
```

Reply:
My bad!